js/territory/typeahead.js

```diff
@@ -9,6 +9,7 @@ define([
     'templates/search/suggestion.hbs',
     'i18n'
 ], function($, API, Bloodhound, header, suggestion, i18n) {
+    const MAX = 2;
```

Comment:
Maybe extend to more than 2 (5?) given that there are more cases than I thought.